core/system/src/mpu/vmpu_freescale_k64_aips.c

```diff
@@ -117,6 +109,16 @@ int vmpu_aips_add(uint8_t box_id, void* start, uint32_t size, UvisorBoxAcl acl)
     return 1;
 }
 
+void vmpu_aips_common_add(void)
+{
+    int box, slot;
+    for (box = 1; box <= g_vmpu_box_count; box++) {
```

Comment:
Thanks @AlessandroA , the fix looks great! Want to be sure regarding this comment:
> `/* contains the total number of boxes`
> `* boxes are enumerated from 0 to (g_vmpu_box_count - 1) and the following`
> `* condition must hold: g_vmpu_box_count < UVISOR_MAX_BOXES */`

If `g_vmpu_box_count ==  UVISOR_MAX_BOXES - 1` (`g_vmpu_box_count < UVISOR_MAX_BOXES` condition), boxes are enumerated from `0` to `UVISOR_MAX_BOXES - 2`. There are `UVISOR_MAX_BOXES` allocated. Is this expected behavior?